rubyforgood/partner
# 1st iteration of issue WIP

1. alphabetically sort puldown
2. Is this the only pulldown I needed to alphabetize?
3. Found appropriate code, and changed it
4. Manually tested it, no problems
5. check to find other pulldowns and see if you agree with ordering
6. ^^
7. Verify if changes were made in correct place/ if more alphabetization is needed



Resolves #001 

### Description


### Type of change



* Bug fix (non-breaking change which fixes an issue)
* New feature (non-breaking change which adds functionality)
* Breaking change (fix or feature that would cause existing functionality to not work as expected)
* This change requires a documentation update
* Documentation update

### How Has This Been Tested?



### Screenshots
